docs/content/latest/architecture/core-functions/universe-creation.md

```diff
@@ -30,9 +30,8 @@ their load, but no tablets would exist in the system yet.
 ## An example
 
 Let us illustrate this with our usual example of creating a 4-node YugabyteDB universe with a
```

Comment:
Not expecting you to fix this, but we shouldn't be using "us" or "we" in the docs. I've been fixing this as time permits, but the convention is that it's OK to use "you" and avoid "we" or "us." When referring to Yugabyte, it can be OK to use we sometimes, but usually it's he's to say something like "Yugabyte supports ..." or "Yugabyte recommends ...".